doc/source/management/data_at_rest_encryption.rst

```diff
@@ -363,33 +704,81 @@ System Variables
 
 .. variable:: keyring_vault_config
 
-  :version 5.7.20-18: Implemented
-  :cli: ``--keyring-vault-config``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Text
-  :default:
+   :version 5.7.20-18: Implemented
+   :cli: ``--keyring-vault-config``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default:
 
 This variable is used to define the location of the
 :ref:`keyring_vault_plugin` configuration file.
 
 .. variable:: keyring_vault_timeout
 
-  :version 5.7.21-20: Implemented
-  :cli: ``--keyring-vault-timeout``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Numeric
-  :default: ``15``
+   :version 5.7.21-20: Implemented
+   :cli: ``--keyring-vault-timeout``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``15``
 
 This variable allows to set the duration in seconds for the Vault server
 connection timeout. Default value is ``15``. Allowed range is from ``1``
 second to ``86400`` seconds (24 hours). The timeout can be also completely
 disabled to wait infinite amount of time by setting this variable to ``0``.
 
-Other reading
--------------
+.. _data-at-rest-encryption.data-scrubbing:
+
+Data Scrubbing
+================================================================================
+
+While data encryption ensures that the existing data are not stored in plain
+form, the data scrubbing literally removes the data once the user decides they
+should be deleted. Compare this behavior with how the ``DELETE`` statement works
+which only marks the affected data as deletable - the space claimed by this data
```

Comment:
nit: deleted data is not "marked as deletable" (something deleted is already past the point of being deletable), but "marked as deleted"

Reply:
Text updated as suggested